.gitignore

```diff
@@ -9,4 +9,9 @@
 *.orig
 *.*#
 *.bak
-
+/mockups
```

Comment:
@sorice I made some changes in your branch . Please merge and ensure you keep those changes i.e. no extra lines in .gitignore